packages/opentelemetry/src/sampler.ts

```diff
@@ -138,22 +137,14 @@ export class SentrySampler implements Sampler {
   }
 }
 
-function getParentRemoteSampled(parentSpan: Span): boolean | undefined {
-  const traceId = parentSpan.spanContext().traceId;
-  const traceparentData = getPropagationContextFromSpan(parentSpan);
-
-  // Only inherit sampled if `traceId` is the same
-  return traceparentData && traceId === traceparentData.traceId ? traceparentData.sampled : undefined;
-}
-
 function getParentSampled(parentSpan: Span, traceId: string, spanName: string): boolean | undefined {
   const parentContext = parentSpan.spanContext();
 
   // Only inherit sample rate if `traceId` is the same
   // Note for testing: `isSpanContextValid()` checks the format of the traceId/spanId, so we need to pass valid ones
   if (isSpanContextValid(parentContext) && parentContext.traceId === traceId) {
     if (parentContext.isRemote) {
-      const parentSampled = getParentRemoteSampled(parentSpan);
+      const parentSampled = getSamplingDecision(parentSpan.spanContext());
```

Comment:
This method implementation as redundant, we already check traceId outside here, so we can simply get the decision directly. This was the only place where we still used `getPropagationContextFromSpan`.